### DISCUSSION TITLE
Image selection / filtering criteria

### DISCUSSION THREAD

**Discussion by codeyls** [containrrr/watchtower#658] [Q&A] [answered]
Hallo,

I'm just getting started, and have read through most of the docs on the watchtower site. One of the pages covers container selection, which I found interesting. Naturally, my next thought was image selection. It would be quite nice if I could configure watchtower to pull the latest image, but only if it meets certain criteria. For example, I'd like to say only update containers in our staging environment if the label "staging-ready" is applied. We're using Harbor for our image repository, if that matters.

Is this possible?

Would appreciate any feedback. Thanks.

**Comment by piksel** [comment, marked as answer]
The easiest way to accomplish this should be to tag the image with `latest-staging` or something similiar and then use `yourapp:latest-staging` as the container image.

**Comment by codeyls** [comment]
Hi piksel,

Thanks for the answer. This makes sense, but will unfortunately not work with our process. When we deploy images, we refer to a unique tag for the release so that we can more easily trace between multiple systems and workflows. If you're taking input for the project, it would be great to see an image filtering capability.

Thanks again.
